cypress-tests/cypress/support/commands.js

```diff
@@ -2365,26 +2310,38 @@ Cypress.Commands.add(
   }
 );
 
-Cypress.Commands.add("stepUp", (stepUpBody, globalState, value) => {
-  const key = `step_up_enabled_${globalState.get("merchantId")}`;
-  stepUpBody.key = key;
-  stepUpBody.value = value;
+Cypress.Commands.add("updateConfig", (configType, configData, globalState, value) => {
+  let url;
+  let body;
+  
+  switch (configType) {
+    case 'autoRetry':
+      url = `${globalState.get("baseUrl")}/configs/autoRetry`;
+      body = { ...configData, enable: value }; // enable or disable autoRetry
```

Comment:
I have a hunch that this would not work as we only pass `key` and `value` in body for all 3 commands: https://github.com/juspay/hyperswitch/blob/main/cypress-tests/cypress/fixtures/configs.json

So you would have to revert back to passing data something like below:
```js
  body = { configData.key = key, configData.value = value };
```